tutorials/schedule-dataflow-jobs-with-cloud-scheduler/schedule-dataflow-jobs-with-cloud-scheduler.md

```diff
@@ -0,0 +1,73 @@
+# Schedule Your Dataflow Batch Jobs With Cloud Scheduler
+
+[Cloud Dataflow](https://cloud.google.com/dataflow) is a managed service for handling 
+both streaming and batch jobs. For your streaming jobs, you just need to launch them once without worrying about operating them afterwards. 
+However, for your batch jobs, you probably need to trigger them based on certain conditions.
+
+In the post, I will show you how you can leverage [cloud scheduler](https://cloud.google.com/scheduler/) to schedule your Dataflow batch jobs. You can find all the code in this [repo](https://github.com/zhongchen/GCP-Demo/tree/master/demos/scheduler-dataflow-demo).
+
+First things first, to be able to run your Dataflow jobs on a regular basis, you need to build your Dataflow templates. 
+Follow the [instructions](https://cloud.google.com/dataflow/docs/guides/templates/creating-templates) to create your templates and save them in a GCS bucket.
+
+![Upload Dataflow templates in a GCS bucket](store_a_template_in_gcs.png)
+
+
+Once you have your templates ready, you can set up cloud schedulers to trigger Dataflow templates.
+
+![Set up your cloud scheduler](set_up_the_cloud_scheduler.png)
+
+
+If you use Terraform, here is one example to define a scheduler.
+
+```hcl-terraform
```

Comment:
If you import the TF into this repo, you can use the includecode directive to pull it in directly: https://g3doc.corp.google.com/devrel/g3doc/infra/guides/devsite.md?cl=head#including-in-devsite

Reply:
I can't verify it locally through the markdown addon. I will stick with the current approach for now.

Reply:
OK, we can change this later, leaving open for now.